### ISSUE TITLE
what is the difference between "d2_ots.pth","d2_tf.pth",and "d2_tf_no_phototourism.pth"?

### ISSUE THREAD

**Issue by ahong007007** [mihaidusmanu/d2-net#43]
Dear Mihai, first of all thanks for your great work.
1.what is the difference between "d2_ots.pth","d2_tf.pth",and "d2_tf_no_phototourism.pth"?
2.Can you open source D2Net based on the ResNet model?

Thank you!

**Comment by mihaidusmanu** [comment]
Hello. `d2_ots.pth` is the Caffe ImageNet pretrained model (off-the-shelf). `d2_tf.pth` is the fine-tuned version of `d2_ots.pth` on the full MegaDepth dataset using the loss from our paper. `d2_tf_no_phototourism.pth` is also a fine-tuned version of `d2_ots.pth` on MegaDepth without the common scenes with PhotoTourism.

As we mentioned in the paper, vanilla ResNets start by rapidly down-sampling the input image which makes them less adequate for this application. Thus we did not investigate them further and we do not have any trained ResNet weights to release. In case you want to use the pretrained weights, then you will have to replace the test model (see below) with a subset of ResNet layers from `torchvision.models.ResNet**`.

https://github.com/mihaidusmanu/d2-net/blob/81983664d1a29fabf09636b3735c5df9151db600/lib/model_test.py#L10-L33

**Comment by ahong007007** [comment]
thank you !
